.docker-compose.datagov-theme.yaml

```diff
@@ -1,8 +1,27 @@
 version: '3.2'
 
 services:
+
+  nginx:
+    depends_on:
```

Comment:
There's no need to repeat everything - just add the stuff that you're modifying (i.e. the ports only)